src/main/java/com/github/fishio/SinglePlayerPlayingField.java

```diff
@@ -69,6 +69,8 @@ public void postTick() {
 				}
 			}
 		});
+		
+		new PowerUpSpawner(this);
```

Comment:
I disagree.Right now the PowerUpSpawner has nothing useful for other classes to interact with. Perhaps later, once some other useful methods in PowerUpSpawner exist, this is a good idea.
Also, storing the PowerUp spawner in a field will make every static analysis tool complain about the field because it's "unused".